### PR TITLE
variable fixes for main

### PR DESCRIPTION
**Pull request by StraussKE**

### CODE REVIEW

File: app/src/main/java/com/example/team42fitness/ui/food/NutritionAdapter.kt [hunk 1]
```diff
@@ -150,15 +153,6 @@ class NutritionAdapter2: Adapter<NutritionAdapter2.NutritionViewHolder2>(){
                         fat.text = n
                     }
 
-                    "Carbohydrate, by difference" -> {
```
Comment (yevgeniychaplygin3) [inline app/src/main/java/com/example/team42fitness/ui/food/NutritionAdapter.kt:153]: Why was this deleted?

Comment (StraussKE) [reply app/src/main/java/com/example/team42fitness/ui/food/NutritionAdapter.kt:153]: That's a very good question

**Review by yevgeniychaplygin3 — APPROVED** [review]
Thanks for doing all that! Sorry again about the naming conflicts.
